common/remote/rmtsmtp.cpp

```diff
@@ -773,77 +800,138 @@ static const char *quit="QUIT\r\n";
 
 static void doSendEmail(CMailInfo & info, CMailPart const & part)
```

Comment:
Not really a change for now, but this would make much more sense as a member function of CMailInfo